src/Commands/stubs/scaffold/provider.stub

```diff
@@ -18,6 +18,7 @@ class $CLASS$ extends ServiceProvider
         $this->registerConfig();
         $this->registerViews();
         $this->registerFactories();
+        $this->registerObservers();
```

Comment:
I would prefer if this was not present by default. Instead, have this documented that it's required to add to have them loaded.

Reply:
Firstly it is useful. Minimizes manual work.

I can say the same thing about `$this->registerViews();` and `$this->registerFactories();`.
In my case, I need observers, instead of views and factories.

Reply:
I understand why it's useful. I'm asking because not a lot of people use these, hence why there's not much demand for this feature.

Do you think more people use observers than views?

Reply:
I only develop a backend. I have only observers. I don't have a view. I just leave empty `registerViews()`, even if I don’t have views. Those who do not have observers can do the same. It costs nothing in performance. It seems like a compromise.

Reply:
I understand your need, I'm asking if you think that's what most people have.

Reply:
I think that people who have grown up to dividing the application into modules have also grown up to using observers. People who do not use observers are likely to not divide applications into modules. I began to use observers more actively when I began to divide the application into modules. With observers it is useful to hang the functionality of one module on the model of another module. The division into modules encourages you to make more servers.

Reply:
I don't use observers.

Reply:
laravel-module exists since 2016, and in 4 years nobody requested anything regarding observers. You're right in the sense that observers are awesome, and maybe everybody should use them, turns out this is not the case so far.

You should consider removing this `$this->registerObservers();` and documenting it. If people come complaning that this is not active by default i'm sure @nWidart will be pleased to turn it on by default.